fvtest/compilertriltest/ArithmeticTest.cpp

```diff
@@ -145,6 +259,68 @@ TEST_P(Int64Arithmetic, UsingLoadParam) {
     ASSERT_EQ(param.oracle(param.lhs, param.rhs), entry_point(param.lhs, param.rhs));
 }
 
+TEST_P(UInt64Arithmetic, UsingConst) {
+    auto param = TRTest::to_struct(GetParam());
+
+    std::string arch = omrsysinfo_get_CPU_architecture();
+    SKIP_IF(param.opcode == "ludiv" && (OMRPORT_ARCH_PPC64 == arch || OMRPORT_ARCH_PPC64LE == arch), MissingImplementation)
+        << "The Power codegen does not yet support ludiv (see issue #2227)";
+
+    char inputTrees[1024] = {0};
+    std::snprintf(inputTrees, sizeof(inputTrees),
+      "(method return=Int64"
+      "  (block"
+      "    (lreturn"
+      "      (%s"
+      "        (lconst %lld)"
```

Comment:
To avoid causing warnings on different compilers, we're trying to replace these string formats with OMR_PRId*:
```suggestion
      "        (lconst %" OMR_PRId64 ")"
```

Reply:
Fixed in eee6b0e06cc180e296222844971d147b048919ee